test/node/features/fixtures/webpack/webpack.config.js

```diff
@@ -8,7 +8,13 @@ module.exports = {
     path: path.resolve(__dirname, 'dist'),
     filename: 'index.bundle.js',
     library: '',
-    libraryTarget: 'commonjs'
+    libraryTarget: 'commonjs',
+  },
+  // The version of terser shipping with webpack 4 does not work in node 20
+  // without the `--openssl-legacy-provider` flag. So we disable minification
+  // in order to continue being able to test webpack 4 on node 20.
+  optimization: {
+    minimize: false,
```

Comment:
any version of webpack 4 on node 20 fails with `Error message "error:0308010C:digital envelope routines::unsupported"` to do with changes to lib ssl version being used by an older version of the terser plugin that comes with webpack. To work around this just disable minification as it's not necessary for our testing